src/audio_core/hle/wmf_decoder_utils.h

```diff
@@ -6,31 +6,21 @@
 // AAC decoder related APIs are only available with WIN7+
 #define WINVER _WIN32_WINNT_WIN7
 
-#include <assert.h>
+#include <iostream>
+#include <string>
+#include <tuple>
 #include <comdef.h>
 #include <mfapi.h>
 #include <mferror.h>
 #include <mfidl.h>
 #include <mftransform.h>
-#include <stdio.h>
-
-#include <iostream>
-#include <string>
-#include <tuple>
 
 #include "adts.h"
 
-enum MFOutputState { FATAL_ERROR = -1, OK = 0, NEED_MORE_INPUT, NEED_RECONFIG, HAVE_MORE_DATA };
+enum MFOutputState { FATAL_ERROR, OK, NEED_MORE_INPUT, NEED_RECONFIG, HAVE_MORE_DATA };
```

Comment:
`enum class`. Also our style for enum uses PascalCase like `FatelError`

Reply:
Changed to `enum class` and changed naming convention